src/background/providers/cloudflare.test.ts

```diff
@@ -0,0 +1,334 @@
+import { jest } from '@jest/globals';
+import { CloudflareProvider } from './cloudflare';
+import { Storage } from '../storage';
+import Token from '../token';
+
+export class StorageMock {
```

Comment:
One first change is to convert `Storage` into a interface, so the mock can be instantiated by class implementing `Storage` interface.